frontend/src/views/RegisterView.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="page_box_wrapper">
+		<div class="page_box">
+			<h1>REGISTER</h1>
+			<form
+				method="post"
+				action=""
+				class="c_block c_form_group"
+				@submit.prevent="userStore.register(registerForm)"
+			>
+				<InputField label="username" v-model="registerForm.name" />
+				<InputField label="password" v-model="registerForm.password" />
+				<InputField
+					label="confirm password"
+					v-model="registerForm.password_confirm"
+				/>
+				<div class="c_block c_split">
+					<p><a href="/login">back</a></p>
+					<p>
+						<input
+							class="link_button"
+							action="/settings"
```

Comment:
the `input` element does not have an `action` attribute in html (does this do something?)

Reply:
haha yes you are very right! I'll remove this, this came from absolute desperation  for an answer